k-distribution/tests/regression-new/freshConfig/Makefile

```diff
@@ -0,0 +1,7 @@
+DEF=test
+EXT=test
+TESTDIR=.
+KOMPILE_FLAGS=--enable-search
+KRUN_FLAGS=--pattern "<k> V:K </k>"
```

Comment:
Are these flags needed for this example? Or copy/paste leftovers?

Reply:
They're needed. I wanted the output to match after checking that each expected fresh constant is distinct, without their actual values being a factor.